src/FrameBuffer.cpp

```diff
@@ -91,7 +91,6 @@ void FrameBuffer::_setAndAttachTexture(u16 _size, CachedTexture *_pTexture)
 
 #ifndef GLES2
 		glTexStorage2D(GL_TEXTURE_2D, 1, fboFormats.colorInternalFormat, _pTexture->realWidth, _pTexture->realHeight);
-		glTexSubImage2D(GL_TEXTURE_2D, 0, 0, 0, _pTexture->realWidth, _pTexture->realHeight, fboFormats.colorFormat, fboFormats.colorType, nullptr);
```

Comment:
All these changes goes into "Use glTexStorage for frame buffer textures." commit